tuner/tuner/dispatch_constraints_test.py

```diff
@@ -37,7 +38,9 @@ def test_generate_solutions(tuner_ctx: common.TunerContext) -> None:
     problem_size = common.ProblemSize(
         matmul_size, lhs_type, rhs_type, res_type, common.DispatchKind.mmt
     )
-    configs = dispatch_constraints.generate_solutions(tuner_ctx.logger, problem_size, 4)
+    configs = dispatch_constraints.generate_solutions(
+        tuner_ctx.logger, problem_size, 4, []
```

Comment:
Doesn't this change the reason for no solutions? Now it can't produce any because there are no MFMAs to choose from.